packages/connector/src/actions.js

```diff
@@ -95,7 +95,7 @@ function cloneAndOpenRepo(cloneURL, tab_id) {
         });
       });
   } else {
-    const command = `git clone ${cloneURL} ${localPath} && code ${localPath}`;
+    const command = `git clone ${cloneURL} ${localPath} && $(which code || which codium) ${localPath}`;
```

Comment:
If the initial program doesn't exist, the command fails. Can you check this once if it is working? Tried it in my PC doesn't seem to work. Instead this should work.
```suggestion
    const command = `git clone ${cloneURL} ${localPath} && code ${localPath} || codium ${localPath}`;
```

What do you think?